charts/gitea-app-operator/templates/rbac.yaml

```diff
@@ -2,35 +2,41 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ include "gitea-operator.serviceAccountName" . }}
+  name: {{ include "gitea-app-operator.serviceAccountName" . }}
   namespace: {{ .Release.Namespace | quote }}
 automountServiceAccountToken: true
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ include "gitea-operator.fullname" . }}-edit-oauth
+  name: {{ include "gitea-app-operator.fullname" . }}-edit-oauth
 rules:
```

Comment:
Could you justify (as a comment in this file) each RBAC rule?

Reply:
Comment lines are added!